src/fluid/fluid.cpp

```diff
@@ -324,11 +319,6 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
   auto ReduceEn = [](MeshData<Real> *md) {
     return ReduceOneVar<Kokkos::Sum<Real>>(md, fluid_cons::energy::name(), 0);
   };
-  auto MaxDensity = [](MeshData<Real> *md) {
-    return ReduceOneVar<Kokkos::Max<Real>>(md, fluid_prim::density::name(), 0);
-  };
-
-  hst_vars.emplace_back(HistoryOutputVar(HstMax, MaxDensity, "maximum density"));
```

Comment:
Looks like this branch might be out of date with main... double check you and @mari2895 are in sync so your PRs don't undo each other.

Reply:
Looks like this is the same as current main, unless I missed something.

Reply:
Why is there a diff showing here then? Odd.

Reply:
Oh I see it now... not sure why it's different from main, don't think I did this myself..

Reply:
maybe just merge main in again... see if that fixes it?

Reply:
@AstroBarker Looks the same on my branch too. Don't see anything different.

Reply:
but why is it deleted? should not be deleted